src/test/resources/TestRun.xml

```diff
@@ -3,9 +3,9 @@
 <suite name="Practice">
     <test name="Test1">
         <classes>
-            <class name="railway.RegisterTest">
+            <class name="railway.ChangePwTest">
```

Comment:
Why do you always edit this one?

Reply:
Because I want to run exactly test case that I choose

Reply:
So when you want to run an any test case you will edit it?